autonomy/chain/mint.py

```diff
@@ -97,3 +99,51 @@ def mint_component(
         raise FailedToRetrieveTokenId(
             "Connection interrupted while waiting for the unitId emit event"
         ) from e
+
+
+def mint_service(  # pylint: disable=too-many-arguments
+    ledger_api: LedgerApi,
+    crypto: Crypto,
+    metadata_hash: str,
+    chain_type: ChainType,
+    agent_ids: List[int],
+    number_of_slots_per_agents: List[int],
+    cost_of_bond_per_agent: List[int],
+    threshold: int,
```

Comment:
So this is a bit confusing for me. We support multiple agents for a service which may or may not have different amount of slots. If the amount of slots are same for all of the agents then the check makes sense, but if all of the agents have different  number of slots than this check does not make sense since the threshold won't make sense in this case. 

For example, take 3 agents a, b and c having 4, 8 and 16 slots. Now if we provide threshold 3 it will make sense for agent a but not for b and c and same goes if we choose threshold according to b and c

Reply:
As @mariapiamo says, "all the agent instances combined"

Reply:
yeah, you have to sum up all the slots before considering the threshold. We can only have just a simple check here because is always up to the creator of a service to design it in a wise way and in a way that the threshold makes sense.

Reply:
Addressed [293ff53](https://github.com/valory-xyz/open-autonomy/pull/1733/commits/293ff53a5f9ad6571e3cc0f6aa8d4cdb554a6521)